app/api/activitylog/helpers.js

```diff
@@ -8,66 +12,158 @@ export const methods = {
   migrate: 'MIGRATE',
 };
 
+const buildActivityLogEntry = builder => ({
+  description: builder.description,
+  action: builder.action || methods.update,
+  beautified: builder.beautified,
+  ...(builder.name && { name: builder.name }),
+  ...(builder.extra && { extra: builder.extra }),
+});
+
+export class ActivityLogBuilder {
+  constructor(data, entryValue) {
+    this.description = entryValue.desc;
+    this.data = data;
+    this.action = entryValue.method ? entryValue.method : methods.update;
+    this.beautified = true;
```

Comment:
Do we actually need this flag? As far as I'm understanding this flag is used for controlling rendering on client side. Can't we beautify all raws, even the raw ones so we don't have the condition and get cleaner code and a more straightforward approach?

Reply:
I've removed the beautified flag adding de RAW mark at the parsing, now all entries have the same structure on the client-side.